Core/Core/Store/Store.swift

```diff
@@ -85,6 +85,7 @@ public class Store<U: UseCase>: NSObject, NSFetchedResultsControllerDelegate {
     private func notify() {
         DispatchQueue.main.async {
             self.eventHandler()
+            self.changes = []
```

Comment:
Only remove the accumulated changes after the event handler is called. This helps ensure two quick changes in the frc don't result in the UI only seeing the second set and crashing.

Reply:
This seems like a good idea.